bigtable-hbase-2.x-parent/bigtable-hbase-2.x-integration-tests/pom.xml

```diff
@@ -36,9 +36,241 @@ limitations under the License.
     <google.bigtable.connection.impl>com.google.cloud.bigtable.hbase2_x.BigtableConnection</google.bigtable.connection.impl>
     <google.bigtable.async.connection.impl>org.apache.hadoop.hbase.client.BigtableAsyncConnection</google.bigtable.async.connection.impl>
     <test.timeout>1800</test.timeout>
+
+    <!-- System property `int-test-2x-deps.mode` selects which dependency configuration is active.
+      It can be be one of 3 states: unset, `cbt-shaded-hbase-unshaded` and `cbt-shaded-hbase-minimal-unshaded`
+      The default (unset) will use unshaded bigtable-hbase adapter and hbase-shaded-client.
+      The other 2 modes will use bigtable-hbase-2x-shaded artifact and either the full hbase-client
+      or a partial hbase-client, that only includes the bare minimum of hbase-client.
+    -->
+
+    <!-- This controls the hadoop version in the `cbt-shaded-hbase-minimal-unshaded` mode -->
+    <minimal-dep-hadoop.version>3.3.6</minimal-dep-hadoop.version>
   </properties>
 
   <profiles>
+    <!-- NOTE: this profile is controlled via `int-test-2x-deps.mode` property -->
+    <profile>
+      <id>cbt-unshaded-hbase-shaded-deps</id>
+      <activation>
+        <property>
+          <!-- On by default -->
+          <name>!int-test-2x-deps.mode</name>
+        </property>
+      </activation>
+
+      <dependencies>
+        <dependency>
+          <groupId>org.apache.hbase</groupId>
+          <artifactId>hbase-shaded-client</artifactId>
+          <version>${hbase.version}</version>
+        </dependency>
+
+        <dependency>
+          <groupId>com.google.cloud.bigtable</groupId>
+          <artifactId>bigtable-hbase-2.x</artifactId>
+          <version>2.14.0-SNAPSHOT</version> <!-- {x-version-update:bigtable-client-parent:current} -->
+          <exclusions>
+            <exclusion>
+              <groupId>org.apache.hbase</groupId>
+              <artifactId>hbase-shaded-client</artifactId>
+            </exclusion>
+          </exclusions>
+        </dependency>
+      </dependencies>
+    </profile>
+
+    <!-- NOTE: this profile is controlled via `int-test-1x-deps.mode` property -->
+    <profile>
+      <id>cbt-shaded-hbase-unshaded-deps</id>
+
+      <activation>
+        <property>
+          <name>int-test-2x-deps.mode</name>
```

Comment:
should this be int-test-1x-deps.mode? Or should the NOTE be int-test-2x-deps.mode?